server/routes/journeys/referral/additional-information/controller.ts

```diff
@@ -8,6 +8,8 @@ export class ReferralAdditionalInformationController {
     res.render('referral/additional-information/view', {
       otherInformation,
       backUrl: 'safer-custody',
+      maxLengthChars: 4000,
+      threshold: '75',
```

Comment:
i just have a thought maybe it's better to have `maxLengthChars or 4000` and `threshold or "75"` in the view.njk?
but maybe not.

Reply:
I thought about it, but it isn't really a view concern, it's setting the values, and I think it makes sense to be more explicit about it in the controller.